vidify/__main__.py

```diff
@@ -23,7 +23,7 @@ def start_gui(config: Config) -> None:
     is put together.
     """
 
-    app = QApplication()
+    app = QApplication(sys.argv)
```

Comment:
This isn't necessary. I already get the arguments and config in the `config` module.

Reply:
I get this without it:
```
Traceback (most recent call last):
File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
"__main__", mod_spec)
File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
exec(code, run_globals)
File "vidify/__main__.py", line 58, in <module>
main()
File "vidify/__main__.py", line 54, in main
start_gui(config)
File "vidify/__main__.py", line 26, in start_gui
app = QApplication()
TypeError: QApplication(List[str]): not enough arguments
```

Reply:
Ah, that can be fixed by using `QApplication([])` then. I had seen it like this in tutorials but didn't know it was only for PyQt5.

Reply:
> Ah, that can be fixed by using `QApplication([])` then. I had seen it like this in tutorials but didn't know it was only for PyQt5.

That gives me:
```
QSettings::value: Empty key passed
QSettings::value: Empty key passed
```

Reply:
How about `QApplication(['vidify'])`? `vidify` there acts as the `argv[0]` and no other arguments passed.

Reply:
> How about `QApplication(['vidify'])`? `vidify` there acts as the `argv[0]` and no other arguments passed.

That works :smile: